Systems/electrical.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<PropertyList>
+
+    <filter>
+        <type>gain</type>
+        <gain>100.0</gain>
+        <input>
+            <property>/systems/electrical/battery-charge-percent</property> <!-- this property ranges from 0.0 to 1.0 despite its name -->
+        </input>
+        <reference>0.0</reference>
```

Comment:
What's the purpose of this `<reference>` element? Since it is subtracted from the input, it doesn't do anything.